config_defaults_inc.php

```diff
@@ -512,10 +512,42 @@
 $g_email_receive_own = OFF;
 
 /**
- * set to OFF to disable email check
+ * Email addresses validation
+ *
+ * Determines if and how email addresses are validated. Possible values for
+ * this config are:
+ * - EMAIL_VALIDATE_HTML5 (default): validate using the pattern given by the
+ *   HTML5 specification for 'email' type form input elements
+ *   {@link http://www.w3.org/TR/html5/forms.html#valid-e-mail-address}
+ *   This is the more versatile and standard option, which should be used in
+ *   most situations.
+ * - EMAIL_VALIDATE_PHP: validate using PHP's built-in method (see
+ *   {@link http://php.net/filter_var} with FILTER_SANITIZE_EMAIL filter
+ *   {@link http://php.net/filter.filters.validate}
+ *   This considers addresses like 'user@domain' (i.e. without a top-level
+ *   domain) as invalid.
+ * - EMAIL_VALIDATE_RFC5322: validate using a fully RFC5322-compliant regex
+ *   (see {@link http://squiloople.com/2009/12/20/email-address-validation/}.
+ *   This probably overkill in most scenarios, but if you need to be fully
+ *   standard-compliant, with this you can.
+ * - EMAIL_VALIDATE_AUTO: let PHPMailer's
+ *   {@link http://phpmailer.github.io/PHPMailer/classes/PHPMailer.html#method_validateAddress validateAddress()}
+ *   method pick the best validation pattern. As of PHPMailer v5.2.9, on recent
+ *   systems, this is generally the same as EMAIL_VALIDATE_RFC5322.
+ * - OFF: disable email validation
+ *
+ * With internet-facing installations, it is strongly advised to use either
+ * the default HTML5 (recommended) or PHP methods (if you don't need to handle
+ * email addresses without a top-level domain).
```

Comment:
I recall that we use to create administrator account as administrator@localhost email.  I assume this will be fine until the user attempts to update administrator user, right?  Should we default to empty email address since it is not valid anyway?